buildroot/share/PlatformIO/scripts/STM32F1.py

```diff
@@ -0,0 +1,8 @@
+Import("env")
+
+# Relocate firmware from 0x08000000 to 0x08007000
+env['CPPDEFINES'].remove(("VECT_TAB_ADDR", 134217728))
```

Comment:
I think this syntax is not recognized by all python version... I had to change that

Reply:
What works in its place?

Reply:
https://github.com/tpruvot/Marlin/commit/c154ac6ea38e2d41b171c71ed86e7cf7d1ba1413#diff-eebec1d1fcda83d96049bdec9dad8952R4

Reply:
From what I read they Bigtreetech copied this from mks_robin. I believe... correct me if I am wrong this is how mks_robin locked / encrypted their bootloader from being flashed. I am not even sure this is needed for Bigtreetech skr mini???